crowdin/messages.json

```diff
@@ -1293,6 +1293,7 @@
   "1760558932": "Apple",
   "1764495771": "How can I close my account?",
   "1765454171": "Commission rate",
+  "1767169921": "Just testing some stuff",
```

Comment:
should we include this? 😅

Reply:
No it's just there for show casing the new key extraction approach, I added a TODO comment in the actual file to remove this before merge :sweat_smile: